### ISSUE TITLE
Find solution for naming (not italicizing)

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#902]
Throughout the site in English and French

Italics does not translate well to screen reader.

How might we keep emphasis on the fact it's the product name?

Difference is important in French where Notification is not always deciphered as a proper noun.

Style that doesn't appear anywhere else
Could not use bold anywhere else?

**notes**
- Can we use bold for Notify?
- Should we rename Notify to GC Notify / GC Notification

https://docs.google.com/document/d/18W-nKAH-54wpNy5XSZeEA31my4XRMxpZRVJ3en4ThAA/edit

https://jamboard.google.com/d/1HGYVkc6Q0Zm5UfH7TaruPy0L5U5JYUIoK-Wl3WyVYJA/viewer?f=0

**Comment by anikbrazeau** [comment]
This change should also apply across FreshDesk and Templates Notify sends..

**Comment by ClementineHahn** [comment]
@anikbrazeau found 500 occurences of Notify to replace, proposed a PR that broke test, Need @sastels  input to fixes test and finalise PR
@anikbrazeau will also look into template of emails today to replace italics also in those

**Comment by ClementineHahn** [comment]
Blocked as long as the test are broken, @anikbrazeau is looking at @sastels for support on unblocking this.

**Comment by ClementineHahn** [comment]
WE went from 80 test failed to only one, close to resolution
